css/styles.css

```diff
@@ -0,0 +1,589 @@
+
+.rabbit-logo{
+    position: relative;
+    left: 218px;
+    top: 3px;
+}
+
+.rabbit-text{
+    position: relative;
+    left: 360px;
+    top: -122px;
+}
+
+
+@font-face {
+  font-family: bebas;
+    src: url('/fonts/bebas.ttf');
+}
+
+@font-face {
+    font-family: ST;
+    src: url('/fonts/StMarie-Thin.otf');
+}
+
+.rabbit-text-span-calibri{
+    font-family: Calibri;
+    color:  #ffb540;
+    font-size: 29pt;
+    letter-spacing: normal;
+}
+
+.rabbit-text-span-bebas{
+    font-family: bebas;
+    color: dimgray;
+    font-size: 27pt;
+    letter-spacing: 2px;
+}
+
+header{
+    background-color: white;
+    top: 2px;
+    z-index: 2;
+    position: fixed;
+    width: 1600px;
```

Comment:
You should think about <main></main> container for your page
Because defining static width everywhere is bad practice
Instead you can define main container width and set width 100% of all children, which have 1600px width,